pkg/kernel/networkservice/connectioncontextkernel/ipcontext/iprule/common.go

```diff
@@ -54,19 +53,28 @@ func create(ctx context.Context, conn *networkservice.Connection, tableIDs *Map,
 			return errors.WithStack(err)
 		}
 
+		tableMap, ok := tableMaps.Load(conn.NetworkServiceEndpointName)
```

Comment:
If a NSC is connected to 2 NSEs, there might be a conflict since the counter will be separated for the 2 connections. A policy for NSE-1 will create a table and rule with table ID 1, and another policy for NSE-2 will select the same table ID.  

I suggest to use the NET NS URL as key instead. I have proposed this change in this PR: https://github.com/networkservicemesh/sdk-kernel/pull/433